ResourceStatusSystem/Command/RemainingDiskSpaceCommand.py

```diff
@@ -0,0 +1,102 @@
+''' RemainingDiskSpaceCommand
+
+  The Command gets the remaining space that is left in a DIRAC Storage Element
+
+'''
+
+from datetime                                                   import datetime
+from DIRAC                                                      import S_OK, S_ERROR, gConfig
+from DIRAC.ResourceStatusSystem.Command.Command                 import Command
+from DIRAC.Core.DISET.RPCClient                                 import RPCClient
+
+__RCSID__ = '$Id:  $'
+
+
+class RemainingDiskSpaceCommand( Command ):
+  '''
+  Uses diskSpace method to get the remaining space
+  '''
+
+  def __init__( self, args = None ):
+
+    super( RemainingDiskSpaceCommand, self ).__init__( args )
+
+    self.rpc = None
+    self.rsClient = None
+
+  def getUrl(self, SE, protocol = None ):
+    """
+    Gets the url of a storage element from the CS.
+    If protocol is set, then it is going to fetch the
+    url only if it uses the given protocol.
+
+    :param SE: String
+    :param protocol: String
+    :return: String
+    """
+    attributes = [ "Protocol", "Host", "Port", "Path"]
+
+    result = ""
+    for attribute in attributes:
+      res = gConfig.getValue( "/Resources/StorageElements/%s/AccessProtocol.1/%s" % ( SE, attribute ) )
```

Comment:
This is not good because you make assumption as of the section name ```AccessProtocol.1```.  Maybe you should rather use the storageElementFactory rather than gConfig to get what you want.